src/main/java/uk/gov/hmcts/reform/locationrefdata/camel/mapper/CourtVenueMapper.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.hmcts.reform.locationrefdata.camel.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.data.ingestion.camel.mapper.IMapper;
+import uk.gov.hmcts.reform.locationrefdata.camel.binder.CourtVenue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Objects.nonNull;
+
+@Component
+public class CourtVenueMapper implements IMapper {
+
+    @Override
+    public Map<String, Object> getMap(Object courtVenue) {
+        CourtVenue courtVenueType = (CourtVenue) courtVenue;
+        Map<String, Object> courtVenueRow = new HashMap<>();
+        courtVenueRow.put("epimms_id", trim(courtVenueType.getEpimmsId()));
```

Comment:
I fill lets have trim annotation in future not now but take a note of this in next jira's